packages/babel-types/test/validators.js

```diff
@@ -32,6 +32,43 @@ suite("validators", function() {
     });
   });
 
+  suite("isCompatTag", function() {
+    it("should handle lowercase tag names", function() {
+      assert(t.react.isCompatTag("div"));
+      assert(t.react.isCompatTag("span"));
+      assert(t.react.isCompatTag("a"));
+      assert(t.react.isCompatTag("input"));
+    });
+
+    it("should handle custom element tag names", function() {
+      assert(t.react.isCompatTag("example-element"));
+      assert(t.react.isCompatTag("flag-icon"));
+      assert(t.react.isCompatTag("plastic-button"));
+      assert(t.react.isCompatTag("img-viewer"));
+    });
+
+    it("rejects empty or null tag names", function() {
+      assert(t.react.isCompatTag(null) === false);
+      assert(t.react.isCompatTag() === false);
+      assert(t.react.isCompatTag(undefined) === false);
+      assert(t.react.isCompatTag("") === false);
+    });
+
+    it("rejects uppercase tag names", function() {
+      assert(t.react.isCompatTag("Div") === false);
+      assert(t.react.isCompatTag("Span") === false);
+      assert(t.react.isCompatTag("A") === false);
+      assert(t.react.isCompatTag("Input") === false);
+    });
+
+    it("rejects leading dash '-'", function() {
+      assert(t.react.isCompatTag("-div") === false);
+      // assert(t.react.isCompatTag("-span") === false);
+      // assert(t.react.isCompatTag("-a") === false);
+      // assert(t.react.isCompatTag("-input") === false);
+    });
+  });
```

Comment:
Test for trailing dash might be nice. That's a use case that fails currently (but would be fixed by this PR) I think.

Reply:
Thanks for taking the time to review on a Saturday @bvaughn 😄  Let me update the PR.

Reply:
When @hzoo tags me, I come running. (In case there's Mario Kart.)